src/main/flight/pid.c

```diff
@@ -756,7 +756,9 @@ int16_t getHeadingHoldTarget() {
 static uint8_t getHeadingHoldState(void)
 {
     // Don't apply heading hold if overall tilt is greater than maximum angle inclination
+    static bool BigAngleDisabledMag=false;
```

Comment:
Please use `lowerCamelCase` for variable names and use spaces around operators to keep the coding style.